stdlib/public/SDK/Foundation/Decimal.swift

```diff
@@ -327,7 +328,6 @@ extension Decimal {
             
             var i = UInt32(0)
             // this is a bit ugly but it is the closest approximation of the C initializer that can be expressed here.
```

Comment:
I think this comment can be removed?

Reply:
\*shrug\* I thought the loop still counted as ugly, but maybe the comment was just referring to the tuple of zeros. @phausler?